cmd/broker/broker_suite_test.go

```diff
@@ -453,6 +461,25 @@ func (s *BrokerSuiteTest) WaitForOperationState(operationID string, state domain
 	assert.NoError(s.t, err, "timeout waiting for the operation expected state %s != %s. The existing operation %+v", state, op.State, op)
 }
 
+func (s *BrokerSuiteTest) GetOperation(operationID string) *internal.Operation {
+	var op *internal.Operation
+	err := s.poller.Invoke(func() (done bool, err error) {
+		op, err = s.db.Operations().GetOperationByID(operationID)
+		if err != nil {
+			return false, nil
+		}
+		return true, nil
+	})
+
+	if dberr.IsNotFound(err) {
```

Comment:
So in case of any error we return with `nil`. Why so complex logic?

Reply:
Don't you think that this would suffice:
`
if err != nil {
   return nil
}
`